custom-graders/README.md

```diff
@@ -93,7 +93,21 @@ $ export CustomGraderPath=$BASE_PATH/programming-assignments-demo/custom-graders
 $ courseraprogramming grade local demo_grader.v1.1 $CustomGraderPath/DemoAssignmentGrader/SampleSubmission/FactoringSampleSubmission partId HxbKF
 ```
 
-**After your docker grader passes all tests in *courseraprogramming*, its time to upload your docker grader via our authoring tools.**
+**After your docker grader passes all tests in *courseraprogramming*, it's time to upload your docker grader.**
+
+Note: it is important to use your course ID (a base64 UUID) and not your course slug (the short name used in the URLs of the course).
+
+```sh
+$ courseraprogramming upload demo_grader.v1.1 my_Course_1D itemId123 partId456
+```
+
+To publish the newly uploaded grader, you can either: (1) navigate to the web-based authoring tools and click "Publish", or alternatively (2) execute:
```

Comment:
@saeta publish has some caveats like it publishes all unsaved changes even if those aren't grader related. Might be good to have 'publish' just within courseraprogramming or with much less detail so that only advanced instructors who know what they're doing use it.